misk-crypto/src/test/kotlin/misk/crypto/CryptoModuleTest.kt

```diff
@@ -42,14 +42,6 @@ class CryptoModuleTest {
     assertThat(testKey).isNotNull()
   }
 
-  @Test
-  fun testAeadIsEnvelopeKey() {
```

Comment:
This was removed since we swapped the wrapping of keys; the actual AEAD returned from a keymanager will be a plain `Aead`, not the explicitly created `KmsEnvelopeAead`.